.github/workflows/update_keycloack.yml

```diff
@@ -0,0 +1,28 @@
+name: Update Keycloak
+env:
+  BRANCH: "Keycloack_scripts"
+  REPO_NAME: educ-grad-tools
+  KC_BASE_URL: ${{ secrets.KC_BASE_URL}}
+  KC_PASSWORD: ${{ secrets.KC_PASSWORD}}
+  KC_USERNAME: ${{ secrets.KC_USERNAME}}
+  KC_TOKEN_URL: ${{ secrets.KC_TOKEN_URL}}
+  KC_REALM_ID: master
+  
+ 
+on:
+  workflow_dispatch:
+
+     
+jobs:
```

Comment:
Use 'environment' property (dev/test/prod) and set up environmental secrets.

Reply:
Fixed